doczrc.js

```diff
@@ -3,7 +3,6 @@ export default {
   description: 'Documentation and examples for redliner',
   typescript: true,
   menu: ['Home', 'Line', 'RedLiner'],
-  dest: './lib',
+  dest: './.docz/dist',
```

Comment:
We can specify the build command in Netlify as `yarn build` (and the publish directory as ./docz/dist) so we don't need to commit the static files